pkgs/development/python-modules/clikit/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib, buildPythonPackage, fetchPypi
+, isPy27, isPy34
+, pylev, pastel, typing, enum34 }:
+buildPythonPackage rec {
+  pname = "clikit";
+  version = "0.2.3";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0zr1s0xhk62p9a6zcp5whvsb27lddyk8gx03k9l8q18jp7y3igbv";
+  };
+
+  propagatedBuildInputs = [
+    pylev pastel
+  ] ++ lib.optional (isPy27 || isPy34) typing
```

Comment:
we don't have Python 3.4 on master anymore, so this shouldn't be needed.

Reply:
My thought was that this is the constraint in the build file and it doesn't hurt to leave it here as documentation/just in case.